apps/rsautl.c

```diff
@@ -112,7 +112,7 @@ int rsautl_main(int argc, char **argv)
         case OPT_HEXDUMP:
             hexdump = 1;
             break;
-        case OPT_RAW:
+        case OPT_RSA_RAW:
```

Comment:
This name change isn't at all related to this PR, and not necessary.  This is `rsautl`, we already know that we're dealing with RSA keys.  Please undo the changes in this file.

Reply:
actually there is a naming conflict (OPT_RAW is used in one of the vxworks header). In order to be able to compile for VxWorks7 this rename was necessary.

OK?

Reply:
Ah!  Ok then